corehq/apps/hqwebapp/templates/hqwebapp/partials/form_list_form.html

```diff
@@ -32,11 +32,12 @@
                          data-bind="css: { error: errors.{{ spec.key }} }">
                         {% if spec.type == "RAW" %}
                             <span data-bind="text: data.{{ spec.key }}"></span>
-                        {% elif spec.type == "TextInput" %}
+                        {% else %}
```

Comment:
Yeah, that's intentional.  I added in `widget.type` and `widget.attrs`, which are sufficient to style standard fields appropriately.  Right now this is only used in one place anyways, I figure if this approach needs to be expanded for future use-cases, we can add in some `elif`s then.